x/bank/client/rest/sendtx.go

```diff
@@ -20,6 +20,7 @@ import (
 // RegisterRoutes - Central function to define routes that get registered by the main application
 func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *wire.Codec, kb keys.Keybase) {
 	r.HandleFunc("/accounts/{address}/send", SendRequestHandlerFn(cdc, kb, cliCtx)).Methods("POST")
+	r.HandleFunc("/broadcast", BroadcastTxRequestHandlerFn(cdc, cliCtx)).Methods("POST")
```

Comment:
Add to endpoint + description to the docs

Reply:
Let's add the `bank/` prefix to keep consistency with the other endpoints from different modules

Reply:
> Add to endpoint + description to the docs

That is WIP

> Let's add the bank/ prefix to keep consistency with the other endpoints from different modules

Agreed